src/support/LookupSummaryAndPolicies.tsx

```diff
@@ -0,0 +1,55 @@
+import { ButtonPrimary } from '@terra-ui-packages/components';
+import React, { Fragment, useState } from 'react';
+import { TextInput } from 'src/components/input';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import { ResourcePolicies } from 'src/support/ResourcePolicies';
+import { ResourceTypeSummaryProps } from 'src/support/SupportResourceType';
+import { SupportSummary } from 'src/support/SupportSummary';
+
+export const LookupSummaryAndPolicies = (props: ResourceTypeSummaryProps) => {
+  const { query } = Nav.useRoute();
+  const [resourceId, setResourceId] = useState<string>(props.fqResourceId.resourceId);
+
+  function submit() {
+    Nav.updateSearch({ ...query, resourceId: resourceId || undefined });
+  }
+
+  return (
+    <>
+      <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'space-between', marginBottom: '1rem' }}>
+        <div
+          style={{
+            color: colors.dark(),
+            fontSize: 18,
+            fontWeight: 600,
+            display: 'flex',
+            alignItems: 'center',
+            marginLeft: '1rem',
+          }}
+        >
+          {props.displayName}
+        </div>
+        <TextInput
+          style={{ marginRight: '1rem', marginLeft: '1rem' }}
+          key={`${props.fqResourceId.resourceTypeName}-input`}
+          placeholder={`Enter ${props.displayName} ID`}
+          onChange={(newResourceId) => {
+            setResourceId(newResourceId);
+          }}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter') {
+              submit();
+            }
+          }}
+          defaultValue={props.fqResourceId.resourceId || ''}
+        />
+        <ButtonPrimary onClick={() => submit()}>Load</ButtonPrimary>
+      </div>
+      {!!props.loadSupportSummaryFn && (
+        <SupportSummary {...props} key={`${props.fqResourceId.resourceTypeName}-summary`} />
```

Comment:
Again, skeptical of the use of `key` here. _If_ it's needed to reinitialize state in these components, that suggests these components may not be properly responding to changes in props. But at first glance, it doesn't _look_ like that's needed here.

Reply:
for some reason it wasn't working and I figured out keys made it work, but now it seems to work without the keys 🤷

Reply:
If the `key` changes, the existing component instance is unmounted and destroyed and a new one created. It's useful in some cases, but usually shouldn't be necessary.

The most frequent use case for `key` is when rendering a list.
https://react.dev/learn/rendering-lists